ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/PackageCommitter.scala

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.kvutils
+
+import java.util.concurrent.Executors
+import com.codahale.metrics.{Counter, Timer}
+import com.daml.ledger.participant.state.kvutils.Conversions.buildTimestamp
+import com.daml.ledger.participant.state.kvutils.DamlKvutils._
+import com.daml.ledger.participant.state.kvutils.committer.{Committer, Context}
+import com.digitalasset.daml.lf.archive.Decode
+import com.digitalasset.daml.lf.data.Ref
+import com.digitalasset.daml.lf.engine.Engine
+import com.digitalasset.daml.lf.language.Ast
+import com.digitalasset.daml_lf.DamlLf.Archive
+import scala.collection.JavaConverters._
+import scala.concurrent.ExecutionContext
+
+private[kvutils] case class PackageCommitter(engine: Engine)
+    extends Committer[DamlPackageUploadEntry, DamlPackageUploadEntry.Builder] {
+
+  private val serialContext =
+    ExecutionContext.fromExecutor(Executors.newSingleThreadExecutor())
```

Comment:
No need to wrap the `ExecutorServer` into an `ExecutionContext`. The `execute` method from `ExecutorService` (really `Executor`) works the same way.

Reply:
:+1: